students/grzegorz_bajorski/lesson_02_flow_control/the_number_of_zero.py

```diff
@@ -0,0 +1,10 @@
+n = int(input())
+number = 0
+
+for _ in range(n):
+    x = int(input())
+    if x == 0:
+        number += 1
+
+print(number)
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.